.travis.yml

```diff
@@ -40,6 +46,7 @@ stages:
   - test
 jobs:
   include:
+    # Build stage
```

Comment:
Feels a bit redundant, given that the next line is `stage: build` :joy:

Reply:
I found this useful while editing / debugging. But yes, redundant. Removed.